src/pybind/mgr/dashboard/controllers/nvmeof.py

```diff
@@ -66,13 +98,28 @@ def get(self, nqn: str):
         )
         @empty_response
         @handle_nvmeof_error
-        def create(self, nqn: str, enable_ha: bool, max_namespaces: int = 256):
-            return NVMeoFClient().stub.create_subsystem(
+        def create(self, nqn: str, enable_ha: bool,
+                   max_namespaces: int = 256, initiators: str = None):
```

Comment:
What about?
```suggestion
                   max_namespaces: int = 256, allowed_initiators: Optional[str] = None):
```

Reply:
I think its fine, to say just `initiators` , as there is nothing as non-allowed ones.